crates/agglayer-storage/src/columns/mod.rs

```diff
@@ -6,6 +6,14 @@ pub enum CodecError {
     #[error(r#"Serialization error: {0}
         This is a critical bug that needs to be reported on `https://github.com/agglayer/agglayer/issues`"#)]
     Serialization(#[from] bincode::Error),
+
+    #[error(r#"Certificate encoded to an empty byte sequence.
+        This is a critical bug that needs to be reported on `https://github.com/agglayer/agglayer/issues`"#)]
+    CertEmpty,
```

Comment:
I would make this generic: 

```suggestion
    #[error(r#"Value encoded to an empty byte sequence.
        This is a critical bug that needs to be reported on `https://github.com/agglayer/agglayer/issues`"#)]
    EmptyValue,
```